bugbug/similarity.py

```diff
@@ -106,6 +111,21 @@ def get_text(self, bug, all_comments=False):
 
         return "{} {}".format(bug["summary"], comments)
 
+    def get_similar_bugs(self, bug):
+        similar_bug_ids = self.get_similar_bugs_child(bug)
+        similar_bugs = bugzilla.get(similar_bug_ids)
```

Comment:
We should avoid making a query here and use the bugs from the DB (`get_similar_bugs_child` will return bugs from the DB anyway, so we are sure they are available).